security_groups/locals.tf

```diff
@@ -0,0 +1,37 @@
+locals {
+  # The ports that the IPA servers should listen on
+  ipa_ports = {
```

Comment:
Don't we need the DNS TCP/UDP ports in this list?

Reply:
Not yet.  I opted to stick with AWS DNS for now, since I would have to rip out the zones that are created in [cisagov/cool-sharedservices-networking](https://github.com/cisagov/cool-sharedservices-networking) in order to get FreeIPA hosting DNS.  Too many things changing at once!

I plan to revisit this later.  There are definite advantages to letting FreeIPA host DNS for hosts that are attached to the domain (automatic DNS updates as IPA servers are added and removed and as hosts join/leave the domain), but I need to verify that it won't interefere with public DNS, for example.

Reply:
Got it - thanks for the explanation! 👍